console/lunox.ts

```diff
@@ -7,20 +7,22 @@ import {
   watch,
   serve,
 } from "./commands/build";
-import { tryCommand, runCommand } from "./commands/runner";
+import { tryCommand } from "./commands/runner";
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const VERSION = greenBright(require("../package.json").version);
 program.version(blue("Lunox Framework ") + "version " + VERSION);
 program.description("Laravel-Flavoured NodeJs framework");
 program.showHelpAfterError(true);
 
+
+
 program
   .command("prod")
   .description("build lunox application for production")
   .action(() => {
     tryCommand("build production", async () => {
-      await runCommand("rm -rf dist");
+      deleteHelper(base_path("dist"));
```

Comment:
we cannot use global helper from `helper.ts`. Its only valid on lunox skeleton project. So I think it's better to redeclare this function in `/commands/build.ts` and we can import it
```js
import {
  bundleTs,
  buildServer,
  buildClient,
  watch,
  serve,
  copyPath,
  deletePath,
} from "./commands/build";
```

Reply:
okey sir